tests/console/zypper_lifecycle.pm

```diff
@@ -63,12 +63,11 @@ sub run {
     # 0755 and all files have 0644 pemmission.
     # For some reason the system will change the permission on /var/cache/zypp/{solv,raw}
     # files. this cause the zypper lifecycle failed when building cache for non-root user.
-    script_run('rm -fr /var/cache/zypp/solv /var/cache/zypp/solv');
-    script_run('rm -fr /var/cache/zypp/raw /var/cache/zypp/raw');
+    assert_script_run('chmod -R u+rwX,og+rX /var/cache/zypp');
 
-    # We add 'zypper ref' here to download and preparse the metadata of packages,
+    # We add 'zypper ref' here to download and parse the metadata of packages,
     # which will make the following 'zypper lifecycle' runs faster.
-    script_run('zypper refresh', 300);
+    script_run('zypper refresh', 420);
```

Comment:
I would say regardless of the next steps, if zypper refresh fails then we shouldn't try to continue the test. If you don't need zypper refresh then delete it

Reply:
@hjluo
Can you please add your comments here? actually I am not sure if any risks to use `assert_script_run`.

Reply:
> @hjluo Can you please add your comments here? actually I am not sure if any risks to use `assert_script_run`.

you can remove it safely here. thanks

Reply:
Thanks, let me delete it and do more VRs